tests/test_strategy_pvpredefined.py

```diff
@@ -333,3 +333,13 @@ def test_correct_interpolation_power_profile():
     times = list(profile)
     for ii in range(len(times)-1):
         assert abs((times[ii]-times[ii+1]).in_seconds()) == slot_length * 60
+
+
+def test_correct_time_expansion_read_arbitrary_profile():
+    market_maker_rate = 30
+    GlobalConfig.sim_duration = duration(hours=3)
+    mmr = read_arbitrary_profile(InputProfileTypes.IDENTITY, market_maker_rate)
+    assert (list(mmr.keys())[-1] - today(tz=TIME_ZONE)).days == 0
+    GlobalConfig.sim_duration = duration(hours=36)
```

Comment:
Optional, but it would be nice to have an additional test for hours=48, to check the border condition at the end of the day.

Reply:
Yes, thanks! Will add this.